easybuild/easyconfigs/f/futhark/futhark-0.19.5.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'Tarball'
+
+name = 'futhark'
+
+version = '0.19.5'
+
+homepage = 'https://futhark-lang.org/'
+
+description = '''Futhark is a small programming language designed to be compiled to efficient parallel code. It 
+   is a statically typed, data-parallel, and purely functional array language in the ML family, and comes 
+   with a heavily optimising ahead-of-time compiler that presently generates GPU code via CUDA and OpenCL, 
+   although the language itself is hardware-agnostic and can also run on multicore CPUs'''
+
+
+toolchain = SYSTEM
+
+source_urls = ['https://futhark-lang.org/releases/']
+sources = ['%(name)s-%(version)s-linux-x86_64.tar.xz']
+checksums = ['4cb1ec38f961ece5e10210f1af45899dd45119141fbeceaba81cb24f3ba55035']
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': []
+}
```

Comment:
Is there a simple command we can run as a sanity check?

Reply:
done